pkg/ccl/backupccl/restore_planning.go

```diff
@@ -1105,14 +1103,12 @@ func restorePlanHook(
 
 	var newDBNameFn func() (string, error)
 	if restoreStmt.Options.NewDBName != nil {
-		if restoreStmt.DescriptorCoverage == tree.AllDescriptors || len(restoreStmt.Targets.Databases) != 1 {
+		if restoreStmt.DescriptorCoverage == tree.AllDescriptors || len(restoreStmt.Targets.
```

Comment:
nit: lets not break a newline on `.` maybe move the whole len clause to the new line. Also the error returned is specific to `RESTORE DATABASE` and multiple databases. I don't think it makes sense to return this error when using `new_db_name` with `RESTORE SYSTEM USERS`.

Reply:
Done.